### PR TITLE
Peg to Girder SHA 7872249a3 Romanesco SHA da82cd17

### PR DESCRIPTION
**Pull request by mgrauer**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -11,6 +11,8 @@ What is Minerva?
 
 Minerva is an open source platform for geospatial visualization using `Girder`_ as its datastore.
 
+Currently Minerva is pegged to Girder version `7872249a3`.
```
Comment (aashish24) [inline docs/index.rst]: So everytime we need to update the docs as well?

**Comment by aashish24** [comment]
Can we not have the version of Girder show up in Minerva (in About dialog or something like that?)

**Comment by mgrauer** [comment]
This PR is to 

1) update the Girder SHA
2) document the Girder SHA
3) open a discussion about how we can peg to Girder

I won't say currently because we don't document it anywhere, but this PR changes the girder SHA in 3 places: travis, our docs, and the ansible playbook for minerva.

What would be ideal is to grab this all from a single location, or fewer locations, and even better if this was officially supported.  I talked to @zachmullen and he is good with adding a `minimum_girder_version` key to the Girder plugin spec.  We would have to implement this in Girder, I think we could do it piggybacking on the work @danlamanna did recently to show which plugins have unmet dependencies, i.e. a message in the Girder plugins page that says "this plugin requires version X of Girder, which is greater than the current version Y" , or some such.

Bringing it back to this PR, ideally we would have it specified in one location and can read that from travis and ansible and possibly readthedocs (it would be easy enough to point people to our plugin.json file), but for now we have to update it in 3 places.

I'll add an issue for an about dialog/hover/display for the Minerva and Girder versions.

**Comment by aashish24** [comment]
Sounds good @mgrauer so is this ready for merge? or you still want to add some more to it? I am trying to see if we can minimize WIP branches :smile:

**Comment by mgrauer** [comment]
@danlamanna brings up good points.

For clarification, @zachmullen has not offered to do this work, but he supports the idea if we implement it.

Beyond the minimum version, we could have a plugin property that is instead "girder_version", and allow ">=, <=, ==, <" to do things like "girder>=1.0,<2.0".  It would be nice to start versioning minerva more actively to make explicit the version of minerva and its relation to the version of girder, though that could potentially complicate matters.

**Comment by mgrauer** [comment]
@aashish24 To me the concern is more about long standing branches hanging out, and I think the bottleneck there is QA and feedback.

This branch needs more work.  I'll definitely update it to drop WIP when ready.

**Comment by danlamanna** [comment]
Curious if @jbeezley has anything to add about plugins pinning Girder to a version since he's been in the versioning trenches a bit.

**Comment by jbeezley** [comment]
I'm still suffering some latent PTSD.  Pinning a to a version is a necessity for CI reliability, but we need to make sure we make it a priority to support the latest version of Girder if not the latest master.  I don't think keeping them in sync should be too much of an issue because most of us actively develop in both places.

Allowing plugins to specify girder versions (particularly in the `<=` direction) worries me.  I think it's generally okay to have a `>=` requirement in the python world, but when you go the other direction you are creating dependency hell for anyone trying you use it.  I think the only sane thing to do is treat a plugin that doesn't work with the most recent version of girder as broken.  If no one gets around to fixing it, then we abandon it.

**Comment by aashish24** [comment]
> This branch needs more work. I'll definitely update it to drop WIP when ready.

Sure, we will try to review sooner and quicker.

**Comment by mgrauer** [comment]
Tested with vagrant/ansible and works.

Ready for review.

**Comment by mgrauer** [comment]
@kotfic This updates Girder SHA for your use.  Can you review? 

and then I can revise #277  accordingly to be based off of this.

**Comment by danlamanna** [comment]
@mgrauer I need this sha of Girder for #289 - https://github.com/girder/girder/commit/e61477c9c50e33a38c39eae0a00c9becca028589

**Comment by mgrauer** [comment]
@danlamanna I updated to your SHA, though this branch name is goofy now.

works on travis, I provisioned with ansible and it works there too.

LGTY?

File: docs/index.rst [hunk 2]
```diff
@@ -11,6 +11,9 @@ What is Minerva?
 
 Minerva is an open source platform for geospatial visualization using `Girder`_ as its datastore.
 
+Currently Minerva is pegged to Girder version `e61477c`.  To update the version of Girder that Minerva
+depends on, update the version here, in `.travis.yml`, and `gider_version` in `ansible/site.yml`.
```
Comment (danlamanna) [inline docs/index.rst]: girder*

**Comment by danlamanna** [comment]
Yes LGTM

**Comment by aashish24** [comment]
yay! One less branch

**Comment by mgrauer** [comment]
thanks for the reminder

:snowflake: :snowboarder: :snowman:

**Pull request merged**